lib/operations/command_v2.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const OperationBase = require('./operation').OperationBase;
+const resolveReadPreference = require('../utils').resolveReadPreference;
+
+class CommandOperationV2 extends OperationBase {
+  constructor(parent, options) {
+    super(options);
+
+    this.ns = parent.s.namespace.withCollection('$cmd');
+    this.readPreference = resolveReadPreference(parent, options);
+
+    // TODO(NODE-2056): make logger another "inheritable" property
+    if (parent.s.logger) {
+      this.logger = parent.s.logger;
+    } else if (parent.s.db && parent.s.db.logger) {
+      this.logger = parent.s.db.logger;
+    }
+  }
+
+  executeCommand(server, cmd, callback) {
```

Comment:
Should we call this `execute` instead of `executeCommand`?

Reply:
I wanted to make it different, at least for now, since its only being used as a pseudo operation (never being used directly). Additionally, it has a different function signature than the subclassed operations.